### DISCUSSION TITLE
Welcome to try our ManiSkill2 with Tianshou

### DISCUSSION THREAD

**Discussion by Jiayuan-Gu** [thu-ml/tianshou#731] [Show and tell]
Thanks for the great RL framework Tianshou! We are going to provide an example of how to use Tianshou to train a vision-based RL policy on our physically realistic environments in ManiSkill2.

We are announcing [ManiSkill2](https://github.com/haosulab/ManiSkill2). ManiSkill2 is a large-scale robotic manipulation benchmark, focusing on learning generalizable robot agents and manipulation skills. It features 2000+ diverse objects, 20 task categories, and a large-scale demonstration set in [SAPIEN](https://sapien.ucsd.edu/), a fully-physical, realistic simulator. The benchmark can be used to study 2D & 3D vision-based imitation learning, reinforcement learning, and motion planning, etc. We invite you to participate in the associated [ManiSkill 2022 challenge](https://sapien.ucsd.edu/challenges/maniskill/2022/) where we will be awarding prizes to the teams who achieve the highest success rates in our environments.

**Comment by destin-v** [comment]
When will this be available?
